src/molecules/formfields/SliderField/Readme.md

```diff
@@ -0,0 +1,34 @@
+### SliderField Example:
+
+```example
```

Comment:
I couldn't get the component to render in the RCL because it needed a value prop, but you could do something like this for the readme:

```
    <SliderField
      sliderValues={
        [ 1000, 5000, 25000, 500000 ]
      }
      input={{
        value: state.value || 5000,
        onChange: (value) => setState({ value })
      }}
    />

```